grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -213,16 +212,24 @@ public final void close(Throwable exception) {
     public final void close(Throwable exception, boolean cancelled) {
         exception = Exceptions.peel(exception);
         final Metadata metadata = generateMetadataFromThrowable(exception);
-        final Status status = GrpcStatus.fromThrowable(exceptionHandler, ctx, exception, metadata);
+        final Status status = exceptionHandler.orElse(GrpcExceptionHandlerFunction.ofDefault()).apply(ctx,
```

Comment:
Q) Should we still need `.orElse(GrpcExceptionHandlerFunction.ofDefault())`?

Reply:
Applied feedback. Thanks